src/main/java/org/folio/utils/UserPermissionsUtil.java

```diff
@@ -63,6 +63,42 @@ public static void verifyUserHasManagePermission(List<String> newAcqUnitIds, Lis
     }
   }
 
+  public static void verifyPaidPermission(List<String> newAcqUnitIds, List<String> currentAcqUnitIds, String acqSataus,
```

Comment:
what is means acqStatus? we can rename to invoiceStatus

Reply:
sure. will do